src/dashboard/src/settings/common.py

```diff
@@ -291,12 +308,15 @@ def get_env_variable(var_name):
 
 # login-related settings
 LOGIN_REDIRECT_URL = '/'
-LOGIN_URL = '/administration/accounts/login/'
+LOGIN_URL = '/Shibboleth.sso/Login'
```

Comment:
We may not need this now nginx is taking care of "am I logged in or not". Also, logout URLs and flow are not fully working yet - so this section can probably be disregarded for now.